src/lookup-table/services/LookupTable.service.ts

```diff
@@ -89,6 +145,25 @@ const LookupTableService = (structure: LookupTableServiceProps) => {
     fieldMaps = [...dbFieldMap, ...fieldMaps].filter(
       ((set) => (obj: any) => (set.has(obj.label) ? false : set.add(obj.label)))(new Set())
     )
+
+    return {
+      onlyInDb: dbFieldMap.filter(_comparer(fieldMaps)),
```

Comment:
I've suggested a simpler way for fieldMaps to be compared (https://github.com/openmsupply/application-manager-server/pull/286#discussion_r612046890)< iI really want to improve readability of our code vs code size.

Reply:
I used the same method but added sticter types that will give code completion and extracted the comparer method to a helper function.

Resolved with cc04c69